api_tests/tokens/test_views.py

```diff
@@ -0,0 +1,277 @@
+import copy
```

Comment:
The structure of the `api_tests` folder has changed: we now use one file per view. (test_token_list, test_token_detail, etc)